site/src/common/blocks/RichTextBlock.tsx

```diff
@@ -117,21 +117,36 @@ const defaultRenderers: Renderers = {
 
 interface RichTextBlockProps extends PropsWithData<RichTextBlockData> {
     renderers?: Renderers;
+    disableLastGutterBottom?: boolean;
 }
 
 export const RichTextBlock = withPreview(
-    ({ data, renderers = defaultRenderers }: RichTextBlockProps) => {
+    ({ data, renderers = defaultRenderers, disableLastGutterBottom }: RichTextBlockProps) => {
         const rendered = redraft(data.draftContent, renderers);
 
         return (
             <PreviewSkeleton title="RichText" type="rows" hasContent={hasRichTextBlockContent(data)}>
-                {rendered}
+                <Wrapper $disableLastGutterBottom={disableLastGutterBottom}>{rendered}</Wrapper>
             </PreviewSkeleton>
         );
     },
     { label: "Rich Text" },
 );
 
+const Wrapper = styled.div<{ $disableLastGutterBottom?: boolean }>`
```

Comment:
We generally name the outer-most element of a component `Root`. 

```suggestion
const Root = styled.div<{ $disableLastGutterBottom?: boolean }>`
```